cmd/gossamer/configcmd_test.go

```diff
@@ -392,6 +392,7 @@ func TestStrToMods(t *testing.T) {
 }
 
 func TestMakeNode(t *testing.T) {
+	t.Skip()
```

Comment:
Do we have an issue open to resolve this?

Reply:
#360